crates/nu-cmd-lang/src/core_commands/use_.rs

```diff
@@ -30,6 +30,10 @@ impl Command for Use {
             .category(Category::Core)
     }
 
+    fn search_terms(&self) -> Vec<&str> {
+        vec!["environment", "import"]
```

Comment:
I think the meaning of `environment` is pretty orthogonal in the shell world.

```suggestion
        vec!["module", "import", "include", "scope"]
```